### PR TITLE
Store project in workSession:

### PR DESCRIPTION
**Pull request by mrgambal**
this improvement could ease further aggregations by WorkSession model

### CODE REVIEW

File: vulyk/models/stats.py [hunk 1]
```diff
@@ -25,6 +29,7 @@ class WorkSession(Document):
     user = ReferenceField(User, reverse_delete_rule=CASCADE, required=True)
     task = ReferenceField(AbstractTask, reverse_delete_rule=CASCADE,
                           required=True)
+    task_type = StringField(max_length=50, required=True, db_field='taskType')
     answer = ReferenceField(AbstractAnswer, reverse_delete_rule=CASCADE)
 
     start_time = DateTimeField(required=True)
```
Comment (mrgambal) [inline vulyk/models/stats.py]: Also I have noticed naming convention is being violated for the whole collection and for both `start_time` and `end_time` fields. Should I possibly rename them?

Comment (dchaplinsky) [inline vulyk/models/stats.py]: Excuse me? Violated where?

Comment (mrgambal) [inline vulyk/models/stats.py]: If you look [here](https://github.com/mrgambal/vulyk/blob/master/vulyk/models/tasks.py#L24), you should notice we used name transformations to keep column and table names in camelCase format to adhere js conventions.
Although in WorkSession model and even [here and below](https://github.com/mrgambal/vulyk/blob/master/vulyk/models/user.py#L20) we have omitted these directives and meta.

Comment (dchaplinsky) [inline vulyk/models/stats.py]: Whatever works for you.

**Comment by dchaplinsky** [comment]
A my tochno etu khujnyu natianem na staruyj bazu?

**Comment by mrgambal** [comment]
@dchaplinsky that's why I have asked you yesterday.
Ok, removing the latest one to make it compliant.

**Comment by mrgambal** [comment]
Do you have any other comments, @dchaplinsky ?

**Comment by dchaplinsky** [comment]
no

**Comment by mrgambal** [comment]
Then, approve and merge it, pls.

**Pull request merged**